### PR TITLE
adds italian moustache that makes you italian

### PR DESCRIPTION
**Pull request by PKPenguin321**
:cl:
add: As it would happen, the chef does not actually have Italian genes, but rather was being influenced by a strange moustache-a.
/:cl:

Closes https://github.com/tgstation/tgstation/pull/35413

Chef starts with it on

### CODE REVIEW

**Comment by Bawhoppen** [comment]
This is the correct implementation of it

**Comment by PKPenguin321** [comment]
wtf my local repo doesnt show any of the existing italian code and pulling doesnt work

File: code/modules/clothing/masks/miscellaneous.dm [hunk 1]
```diff
@@ -40,6 +40,60 @@
...
+		message = replacetext(message,"spicy","a-spicy")
+		message = replacetext(message," friend "," enemy ")
+		message = replacetext(message," enemy "," friend ")
+		message = replacetext(message,"traitor","mafioso")
+		message = replacetext(message," operative "," greek ")
+		message = replacetext(message," op "," greek ")
+		message = replacetext(message," ops "," greeks")
+		message = replacetext(message," operative "," greek")
+		message = replacetext(message," operatives "," greeks")
+		message = replacetext(message," ops "," greeks")
```
Comment (epochayur) [inline code/modules/clothing/masks/miscellaneous.dm]: did you replace " ops " twice?

File: code/modules/clothing/masks/miscellaneous.dm [hunk 2]
```diff
@@ -40,6 +40,59 @@
...
+		message = replacetext(message," meat",pick(" pepperoni"," prosciutto"))
+		message = replacetext(message,"hello",pick("ciao","buongiorno"))
+		message = replacetext(message," bye ",pick(" ciao "," arrivederci "))
+		message = replacetext(message,"thanks","grazie")
+		message = replacetext(message," wine"," vino")
+		message = replacetext(message,"spaghetti", "SPAGHETT")
+		for(var/i in 1 to length(message))//append -a to about 50% of the words
+			if(message[i] == " " && i > 1 && prob(50))
+				if((65 <= text2ascii(message[i-1]) <= 90) || (97 <= text2ascii(message[i-1]) <= 122))//if the previous character is a letter, upper or lowercase
+					replacetext(message[i], " ", "-a ")
```
Comment (PKPenguin321) [inline code/modules/clothing/masks/miscellaneous.dm]: apparently there's a better way of doing this via regex that im currently struggling to wrap my head around

Comment (PKPenguin321) [reply code/modules/clothing/masks/miscellaneous.dm]: also this doesn't function and im gay

**Comment by ShizCalev** [comment]
closes #35416 too.

**Comment by PKPenguin321** [comment]
will fix and finish in maybe like an hour

**Comment by Potato-Masher** [comment]
Somebody is going to forcefully steal this mustache for themselves, and sadly nobody is going to care enough to stop it.

**Comment by DaedalusGame** [comment]
make this the default mask for the chef

**Comment by PKPenguin321** [comment]
That's on the todo list

**Comment by PKPenguin321** [comment]
I think it should work

**Comment by coiax** [comment]
What about Plasmamen chefs?

**Comment by DaedalusGame** [comment]
they cook themselves first

**Comment by Ordonis** [comment]
You do realize this will only lead to berets that make people speak French. 
Aaand now I have to make French.
But this is a better implementation of-a his-a moustache, and a good model going forward. The only thing I would suggest/request is that keeping genetic possibility of mutating to speak-a only Italiano would-a be nice. A lot of the var edits and defines were specifically for that possibility to be added. If people don't want to go down that road, or would prefer it gets put in brain damage or something, I'd be fine with that too. A costume crate of language changing items could be quite the prize for the Clown to beg for and then not get. He will live in agony knowing that he would have-a to steal this-a moustache and-a take off his-a own mask to speak-a Italian! Also I'm stealing the capitalization thing.

**Comment by sawrge** [comment]
tabernak

**Comment by Ordonis** [comment]
Quebecois detected. Dying surrender flags with maple syrup and less surrendering.

**Comment by PKPenguin321** [comment]
>What about Plasmamen chefs?

do they not override job specific equipment already

**Comment by ChuckTheSheep** [comment]
Make "the" la or il, or one of the other 6 ways to say "the" in Italian.

**Review by Cyberboss — APPROVED** [review]

**Comment by PKPenguin321** [comment]
Pls no my GBP

**Comment by arsserpentarium** [comment]
Why did you delete italian genes?
It shouldn't be roundstart. But why not make  mutation with this trait?

**Comment by PKPenguin321** [comment]
Because it does not need to be a trait, much less a mutation, when it should really only be getting used on this mask and the mask has all the speech override code necessary already

**Pull request merged**